src/components/howTos.stories.mdx

```diff
@@ -8,7 +8,7 @@ You can refer to the usage and some examples below.
 
 In an Electron application, you should create a custom screen picker to retrieve the screen's source ID and pass it to the `toggleContentShare` method from `useContentShareControls` hook to start sharing your screen.
 
-```jsx
+```tsx
```

Comment:
We have JS SDK and component examples in JavaScript. If you look at other pages in story book, you will find they are all written in JSX

Reply:
Thank you for pointing this out. You're right. I will change them to jsx.